source/location_access/locacc_zoras_domain.cpp

```diff
@@ -183,7 +183,7 @@ void AreaTable_Init_ZorasDomain() {
                 }, {
                   //Exits
                   Entrance(ZD_BEHIND_KING_ZORA,       {[]{return true;}}),
-                  Entrance(JABU_JABUS_BELLY_ENTRYWAY, {[]{return (IsChild && Fish);},
+                  Entrance(JABU_JABUS_BELLY_ENTRYWAY, {[]{return (IsChild && Fish) || OpenJabu;},
```

Comment:
```suggestion
                  Entrance(JABU_JABUS_BELLY_ENTRYWAY, {[]{return IsChild && (Fish || OpenJabu);},
```
Oh also I think this needs to be done. Otherwise glitchless logic will expect Adult to be able to enter Jabu when this option is on.